assignments/closure.js

```diff
@@ -1,11 +1,30 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
 
+function publicLib(){
+  let booksPublic = '100 books';
+  console.log(`Public Library has ${booksPublic}`);
+
+    function personalLib(){
+       let booksPersonal = '20 books';
+
+       console.log(`Public library has ${booksPublic} and my personal library has ${booksPersonal}`);
+
+    }
+
+  return personalLib();
+
+}//end of publicLib
+
 
 // ==== Challenge 2: Create a counter function ====
 const counter = () => {
   // Return a function that when invoked increments and returns a counter variable.
+
+   	var count = 0;
+   	return function () {return count+=1;}
```

Comment:
great job here.  One thing to consider.  look into what count++ would do, vs if you had used ++count